scripts/run_multiple_configurations.py

```diff
@@ -194,6 +195,34 @@ def make_configurations(base_config, changing_configurations, settings):
 settings = raw_config["Multiple Configurations Settings"]
 changing_configurations = raw_config["Multiple Configurations Values"]
 
+# Detect the possible configuration combinations of the "Multiple Configurations
+# Settings" and store them in the right order. This information will be written
+# in the run_combination.yaml file as a comment, so that the rename_run_folders.sh
+# can easily parse the comments and rename the run folders automatically.
+base_model_dir = base_config['Logging']['model_directory']
+base_scripts_dir = base_config['Logging'].get('scripts_directory', '')
+multiple_config_values_all = raw_config["Multiple Configurations Values"]
+multiple_config_values = []
+for key in multiple_config_values_all.keys():
+    values = []
+    for val in multiple_config_values_all[key]['values'].keys():
+        values.append(val)
+    if values not in multiple_config_values:
+        multiple_config_values.append(values)
+multiple_config_values = np.array(multiple_config_values)
+
+configuration_combinations = []
+command = "itertools.product("
+for i in np.arange(multiple_config_values.shape[0],0,-1):
+    if i-1 != 0:
+        command += "multiple_config_values[{}],".format(i-1)
+    else:
+        command += "multiple_config_values[{}])".format(i-1)
+
+for combination in eval(command):
+    configuration_combinations.append(combination)
+configuration_combinations = np.array(configuration_combinations)
```

Comment:
Lines 205-224 could be simplified to

```python
multiple_config_values = []
for settings in multiple_config_values_all.values():
    values = tuple(settings['values'])
    multiple_config_values.append(values)
multiple_config_values = list(set(multiple_config_values)) # Remove duplicates
configuration_combinations = itertools.product(*reversed(multiple_config_values))
```

This avoids using an `eval` statement.